Make some arrays const.

pfusik opened the pull request:
While reading cc65 sources I noticed variable arrays that could be const.

oliverschmidt commented on src/cc65/codegen.c:
```diff
@@ -3546,7 +3546,7 @@ void g_ne (unsigned flags, unsigned long val)
 {
-    static const char* ops[12] = {
+    static const char* const ops[4] = {
         "tosltax", "tosultax", "toslteax", "tosulteax",
```
I guess you want the final ',' removed here too.

oliverschmidt commented on src/grc65/main.c:
```diff
@@ -226,18 +226,15 @@ static void openSFile (void)
-    while (strlen (tokenTbl[a]) != 0) {
-        if (strcmp (tokenTbl[a], token) == 0) break;
-        a++;
+    int i;
```
Please keep the blank line separating the definitions from the code.

oliverschmidt commented on src/grc65/main.c:
```diff
@@ -226,18 +226,15 @@ static void openSFile (void)
-        a++;
+    int i;
+    for (i = 0; tokenTbl[i][0]; i++) {
+        if (strcmp (tokenTbl[i], token) == 0) return i;
```
Please put 'return' into own {} block (and own line).

oliverschmidt commented on src/grc65/main.c:
```diff
@@ -226,18 +226,15 @@ static void openSFile (void)
-    if (strlen (tokenTbl[a]) == 0) a = -1;
-    return a;
+    return -1;
 }
```
Please elaborate why you changed the code above beyond the scope of this pull request to add 'const'.

pfusik commented:
Added a commit with the suggested edits.

I changed a parameter type of `findToken` and I found this short function overcomplicated, so refactored it to be easier to read.

oliverschmidt commented:
Thanks for the contribution - and the explanation :-)
